### DISCUSSION TITLE
Where is the documentation for zbm + bullseye?

### DISCUSSION THREAD

**Discussion by LadyMarmelade** [zbm-dev/zfsbootmenu#637] [Questions] [answered]
using zol since many years, i am in the process of setting up an rpool (for the first time). For reasons outside of zfs, i am interested to install oldstable (bullseye), but apparently, all the relevant documents have been removed. I guess, there is a reason for that. But since i am lagging behind by almost 6 years and stable currently does not support features i need, i am searching for documents and hints concerning bullseye.
Where can i find "bullseye-uefi" document, and/or "Debian-Bullseye-installation-with-ESP-on-the-zpool-disk"?
And what do have need to know about the reasons for removal of that documentation?
Anyone with some advanced knowledge can please hint me forward?

**Comment by zdykstra** [comment, marked as answer]
Bullseye documentation was removed when Bookworm was released. It's available as a series of raw documents in the project history. It was removed sometime in the last quarter of 2023.

There's not a lot of differences between Bookworm and Bullseye, so the guide that's live now should generally work.

**Reply by LadyMarmelade** [reply]
Thank you. I have not been able to turn up the full documentation, but found a hint saying, the compatibility settings will not work on bullseye and thus pool definition has to be done manually.
Quite an important detail, i would think.

**Reply by zdykstra** [reply]
Not really - you'd see immediately that the option wouldn't be accepted by `zpool` and you'd just have to drop the argument.

**Comment by no-usernames-left** [comment]
[Bullseye will be EOL around July of this year](https://wiki.debian.org/DebianReleases#Production_Releases). Is there something bookworm is lacking for your use case?

**Reply by LadyMarmelade** [reply]
tx 4 pointing this out.
yes, unfortunately, several gnome extensions necessary for handicapped people no longer work on bookworm, so i may just disconnect from the net and use debian 11 for some more, only dual booting to bookworm if needed.

**Reply by no-usernames-left** [reply]
A non-Debian distro may be a better choice.

**Reply by LadyMarmelade** [reply]
Thank you for suggesting it.
Just today, i managed to have zbm start a bookworm-KDE install, and i am very happy ( + grateful ).
Such a great tool! Thank you everyone. :-)
